charts/k8s-service/README.md

```diff
@@ -1218,3 +1221,39 @@ You can learn more about using private registries with Kubernetes in [the offici
 documentation](https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry).
 
 back to [root README](/README.adoc#day-to-day-operations)
+
+## How to enable Vertical Pod Autoscaler ?
+
+[Vertical Pod Auto scaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler) is used 
```

Comment:
Small grammar check: `is used dynamically change` to `is used to dynamically change`